.template/addons/bootstrap/application.scss.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+insert_into_file 'app/assets/stylesheets/application.scss', after: /@import 'variables'.+\n/ do
```

Comment:
What do you think to import bootstrap in `Dependencies` section?
IMO bootstrap is dependencies instead of variables.

Reply:
We could do that but the problem is I couldn't find a way where `insert_into_file` can detect the `Dependencies` comment and insert after that. I could add `// Dependencies` here but that would make it redundant having two `// Dependencies` in a file. It was bothering me too. What are your thoughts? 🤔

Reply:
Would not be good if we have 2 `Dependencies` comment. I have no idea either about how to detect it.
@andyduong1920 @olivierobert Do you have any idea?

Reply:
@rafayet-monon How about `insert_into_file 'app/assets/stylesheets/application.scss', after: /\/\/ Dependencies\n/ do` cc @gutakk 

as the `// Dependencies` is in that file already

![image](https://user-images.githubusercontent.com/11751745/95566823-4b2d8a00-0a4c-11eb-8404-bd5e771ff8fd.png)

Reply:
It works 🎉 Changed in [ad2d080](https://github.com/nimblehq/rails-templates/pull/239/commits/ad2d0809ae33f68a296cc205772abb8ba00c7482)